product_docs/docs/pgd/5/reference/catalogs-internal.mdx

```diff
@@ -80,51 +80,112 @@ An internal catalog table with per node group configuration options.
 
 Per node group routing configuration options.
 
+#### `bdr.node_group_routing_config_summary` columns
+
+| Name                    | Type    | Description                                                             |
+|-------------------------|---------|-------------------------------------------------------------------------|
+| node_group_name         | name    | Node group name                                                         |
+| location                | name    | Node group location                                                     |
+| enable_proxy_routing    | boolean | Group proxy routing enabled?                                            |
+| node_group_type         | text    | Node group type (one of "global", "data", "shard" or "subscriber-only") |
```

Comment:
One should never see "shard" in actual deployments, that's just in-development stuff that will change before it's released

Reply:
```suggestion
| node_group_type         | text    | Node group type (one of "global", "data",  or "subscriber-only") |
```

Reply:
Will fix.